### PR TITLE
Added debt reminder history

### PR DESCRIPTION
**Pull request by Hopekodi**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: app/Http/Controllers/DebtorController.php [hunk 1]
```diff
@@ -227,14 +227,29 @@ public function destroy(Request $request, $id)
     {
     }
 
+<<<<<<< HEAD
```
Comment (Jeremiahiro) [inline app/Http/Controllers/DebtorController.php:230]: you did not resolve merge conflict

Comment (Douglasokolaa) [inline app/Http/Controllers/DebtorController.php:230]: There is an unresolved merge conflict here

File: app/Http/Controllers/DebtorController.php [hunk 2]
```diff
@@ -227,14 +227,29 @@ public function destroy(Request $request, $id)
...
+        $store_ref_id = explode('-', $id)[0];
+        $customer_ref_id = explode('-', $id)[1];
+        $ts_ref_id = explode('-', $id)[2];
+
+
         $_id = $request->transaction_id;
         $message = $request->message;
 
 
+<<<<<<< HEAD
```
Comment (Douglasokolaa) [inline app/Http/Controllers/DebtorController.php:248]: here too

File: app/Http/Controllers/DebtorController.php (hunk 1 above)
Comment (Hopekodi) [reply app/Http/Controllers/DebtorController.php]: Owk...

**Pull request merged**